.github/actions/build-extension-package/Dockerfile

```diff
@@ -0,0 +1,54 @@
+ARG UBUNTU_VERSION=22.04
```

Comment:
This is imitating this dockerfile, less the not applicable parts 

https://github.com/postgresml/postgresml/blob/master/pgml-extension/Dockerfile

Reply:
The purpose of this file is to build an .deb (package) file for ubuntu with a PGX extension in it

Reply:
will all of our CoreDB postgres images be ubuntu based then, or just the images that contain extensions which we author?

Reply:
I think we have to pick one distro to do first. For example, postgres ML is only doing ubuntu or inside a container (or build from source): https://postgresml.org/user_guides/setup/v2/installation/#install-the-extension